src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/dataset.xml

```diff
@@ -0,0 +1,80 @@
+  <ddm:profile>
+    <dc:title>Reis naar Centaur-planetoïde</dc:title>
+    <dc:title>Trip to Centaur asteroid</dc:title>
+    <dcterms:description>Een tweedaagse reis per ruimteschip naar een bijzondere planetoïde in de omgeving van Jupiter.</dcterms:description>
+    <dcterms:description>A two day mission to boldly go where no man has gone before</dcterms:description>
+    <dcx-dai:creatorDetails>
+      <dcx-dai:author>
+        <dcx-dai:titles>Captain</dcx-dai:titles>
+        <dcx-dai:surname>Kirk</dcx-dai:surname>
+        <dcx-dai:initials>J.T.</dcx-dai:initials>
+        <dcx-dai:organization>
+          <dcx-dai:name xml:lang="en">United Federation of Planets</dcx-dai:name>
+        </dcx-dai:organization>
+      </dcx-dai:author>
+    </dcx-dai:creatorDetails>
+    <ddm:created>2015-05-19</ddm:created>
+    <ddm:audience>D30000</ddm:audience>
+    <ddm:accessRights>OPEN_ACCESS</ddm:accessRights>
+  </ddm:profile>
+  <ddm:dcmiMetadata>
+    <dcterms:publisher>dans</dcterms:publisher>
+    <dcterms:publisher>knaw</dcterms:publisher>
+    <dc:identifier>ds1</dc:identifier>
+    <dc:language>NL</dc:language>
+    <dc:language>encoding=UTF-8</dc:language>
+    <dcterms:rightsHolder>jan</dcterms:rightsHolder>
+    <dcterms:rightsHolder>eko</dcterms:rightsHolder>
+    <dcterms:rightsHolder>linda</dcterms:rightsHolder>
+    <ddm:available>1992-07-30</ddm:available>
+    <dc:format>video/mpeg</dc:format>
+    <dc:format>text/plain</dc:format>
+    <dcterms:spatial>here</dcterms:spatial>
+    <dcterms:spatial>there</dcterms:spatial>
+    <dcterms:alternative>Gewoon een stukje steen in de ruimte?</dcterms:alternative>
+    <dc:source>whatever</dc:source>
+    <dcterms:type>random test data</dcterms:type>
+    <dcterms:replaces>http://x</dcterms:replaces>
+    <dcterms:isRequiredBy>blabla</dcterms:isRequiredBy>
+    <dc:relation>http://y</dc:relation>
+    <dcx-dai:contributorDetails>
+      <dcx-dai:author>
+        <dcx-dai:initials>A.S.</dcx-dai:initials>
+        <dcx-dai:surname>Terix</dcx-dai:surname>
+        <dcx-dai:DAI>info:eu-repo/dai/nl/123456789X</dcx-dai:DAI>
+        <dcx-dai:titles>Prof. Dr. Mr. Ir.</dcx-dai:titles>
+        <dcx-dai:organization>
+          <dcx-dai:name xml:lang="en">Gallië</dcx-dai:name>
+        </dcx-dai:organization>
+      </dcx-dai:author>
+    </dcx-dai:contributorDetails>
+    <dcx-dai:contributorDetails>
+      <dcx-dai:author>
+        <dcx-dai:initials>O.</dcx-dai:initials>
+        <dcx-dai:surname>Belix</dcx-dai:surname>
+        <dcx-dai:insertions>van</dcx-dai:insertions>
+      </dcx-dai:author>
+    </dcx-dai:contributorDetails>
+    <dc:subject>astronomie</dc:subject>
+    <dc:subject>ruimtevaart</dc:subject>
+    <dc:subject xsi:type="abr:ABRcomplex">IX</dc:subject>
```

Comment:
Did some archaeological digging in the git-commits. The definitions needed for the field were introduced  with the start of the life science deposit format and actual usage in a commented out state too.
https://github.com/DANS-KNAW/easy-app/commit/d9b1af3b52230998486e02bb602fd3e0beec8f5d 

It looks like proper spelling should be CAB anyway
https://github.com/DANS-KNAW/easy-app/commit/d9b1af3b52230998486e02bb602fd3e0beec8f5d#diff-f0af4005cd2d9a956e25ed4b0387335aR4